observability/distr_tracing/distr-tracing-tempo-updating.adoc

```diff
@@ -6,6 +6,11 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
+[TIP]
+====
+For upgrading the deprecated and soon to be removed {JaegerShortName}, see xref:../../observability/distr_tracing/distr_tracing_jaeger/distr-tracing-jaeger-updating.adoc#dist-tracing-jaeger-updating[Upgrading] in the {JaegerShortName} documentation.
```

Comment:
> the deprecated and soon to be removed

Reply:
Yes, because Tempo is the replacement product for Jaeger, so readers can confuse the docs and may be looking at the wrong product.